packages/core/src/plugins/web3FormAssets/web-3-form.css

```diff
@@ -0,0 +1,12 @@
+input,
```

Comment:
Hi @yucheng11122017, these CSS selectors are too generic, it causes any input etc elements to have unwanted effects. The most obvious one that caught my attention is the increased in margin for our header. See the preview:
![image](https://user-images.githubusercontent.com/41845017/227410354-a6f4c238-c69e-4f5f-b9bd-17d68343dc5e.png)

Current working version is:
![image](https://user-images.githubusercontent.com/41845017/227410433-e82932f6-740f-407a-a6bf-c6e74296df1a.png)